doc/api/crypto.markdown

```diff
@@ -1153,6 +1168,15 @@ keys:
 
 All paddings are defined in the `constants` module.
 
+### crypto.timingSafeEqual(a, b)
+
+Returns true if a is equal to b, without leaking timing information that would
```

Comment:
`a` and `b` here should have backticks for consistency.

Also, it may be useful to include a note about how the function performs its comparison, that it compares SHA256 HMACs of the inputs.